src/styles/creators.scss

```diff
@@ -0,0 +1,114 @@
+.creators {
+  margin-bottom: 100px;
+  position: relative;
+
+  @include tablet {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 30px;
+    margin-bottom: 130px;
+  }
+
+  @include desktop {
+    max-width: 1280px;
+    margin-bottom: 200px;
+    grid-gap: 120px;
+
+  }
+
+  @include big-desktop {
+    max-width: 1440px;
+  }
+
+  &__description {
+    display: flex;
+    flex-direction: column;
+
+    @include desktop {
+      padding-left: 100px;
+    }
+  }
+
+  &__title {
+    max-width: 280px;
+    position: relative;
+
+    @include tablet {
+      max-width: 100%;
+    }
+
+    @include desktop {
+      max-width: 280px;
+      margin-bottom: 34px;
+    }
+  }
+
+  &__paragraph-last {
+    display: none;
+
+    @include desktop {
+      display: block;
+    }
+  }
+
+  &__paragraphs {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    max-width: 280px;
+    margin-bottom: 30px;
+
+    @include tablet {
+      max-width: 100%;
+      gap: 15px;
+    }
+
+    @include desktop {
+      gap: 30px;
+      max-width: 380px;
+      margin-bottom: 20px;
+    }
+  }
+
+  &__link {
+    margin-bottom: 36px;
+
+    @include desktop {
+      margin-bottom: 50px;
+      width: 86px;
+    }
+  }
+
+  &__button {
+    width: 100%;
+
```

Comment:
The width of the '.creators__button' class is set to 100% for mobile and to a specific pixel value for tablet and up. This is alright if you want your button to take the full width only on mobile. However, if you want it to take the full width on all screens, you can simply set the width to 100% outside the media queries and remove it from the 'tablet' media query.